_integration-schemas/ms-teams/v1/users.md

```diff
@@ -0,0 +1,56 @@
+---
+tap: "ms-teams"
+version: "1"
+key: ""
+
+name: "users"
+doc-link: "hhttps://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-beta"
+singer-schema: "https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/schemas/users.json"
+description: |
+  The `{{ table.name }}` table contains information about users in your Microsoft account.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "List users"
+    doc-link: "https://docs.microsoft.com/en-us/graph/api/user-list?view=graph-rest-beta&tabs=http"
```

Comment:
```suggestion
    doc-link: "https://docs.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http"
```

[This endpoint actually uses the V1 API](https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/streams.py#L121)